docs/configuration.md

```diff
@@ -378,30 +364,12 @@ You can also change the default selection using the [`include`](settings.md#incl
 
 ## Jupyter Notebook discovery
 
-Ruff has built-in support for [Jupyter Notebooks](https://jupyter.org/).
+Ruff has built-in support for linting and formatting [Jupyter Notebooks](https://jupyter.org/).
 
 !!! info
-    Notebooks are linted and formatted by default when using [preview mode](preview.md).
-    You can opt-out of notebook linting and formatting by adding `*.ipynb` to [`extend-exclude`](settings.md#extend-exclude).
-
-To opt in to linting and formatting Jupyter Notebook (`.ipynb`) files, add the `*.ipynb` pattern to
-your [`extend-include`](settings.md#extend-include) setting, like so:
-
-=== "pyproject.toml"
-
-    ```toml
-    [tool.ruff]
-    extend-include = ["*.ipynb"]
-    ```
-
-=== "ruff.toml"
-
-    ```toml
-    extend-include = ["*.ipynb"]
-    ```
-
-This will prompt Ruff to discover Jupyter Notebook (`.ipynb`) files in any specified
-directories, then lint and format them accordingly.
+    Notebooks are linted and formatted by default from Ruff version `0.6.0` onwards which marks
+    Jupyter Notebook support as stable. You can opt-out of linting and formatting notebooks by
+    adding `*.ipynb` to [`extend-exclude`](settings.md#extend-exclude).
```

Comment:
I wonder if we should change the note to: For versions older than `0.6.0` linting and formatting requires explicit opt-in by adding `*.ipynb` to [`extend-include`]

And we may want to add an example of how to exclude notebooks altogether.

Reply:
Looking at the ecosystem results, an example for how to disable some rules for notebooks only might help adoption.

Reply:
These are good suggestions. Let me make some edits.